compute/cloud-client/src/test/java/compute/windows/osimage/WindowsOsImageIT.java

```diff
@@ -51,10 +52,15 @@ public class WindowsOsImageIT {
 
   private static final String PROJECT_ID = System.getenv("GOOGLE_CLOUD_PROJECT");
   private static final String ZONE = "us-central1-a";
+  private static final int MAX_ATTEMPT_COUNT = 3;
+  private static final int INITIAL_BACKOFF_MILLIS = 300000; // 5 minutes
   private static String INSTANCE_NAME;
   private static String DISK_NAME;
   private static String IMAGE_NAME;
   @Rule
+  public final MultipleAttemptsRule multipleAttemptsRule = new MultipleAttemptsRule(
+      MAX_ATTEMPT_COUNT,
+      INITIAL_BACKOFF_MILLIS);
```

Comment:
Is there a purpose for the backoff time? This could make the test time very long

Reply:
Certain resources take an extra minute or two to completely get destroyed. Also, in case of Service unavailable issues or other transient problems, backoff seemed like an option to avoid those.

Reply:
But we can decrease the backoff time. wdyt?